### ISSUE TITLE
Problem when sending model with nested mongoose schema

### ISSUE THREAD

**Issue by erezz33** [giddyinc/mongoose-to-swagger#14]
for instance:
```
const m2s = require('mongoose-to-swagger');
const temp =  mongoose.model('test' , new mongoose.Schema( {   
    a: 
        new mongoose.Schema({
            b: {
                type :  new mongoose.Schema({})
            }
    })
}));
m2s(temp);
```
returns error
but writing this way will work:
```
const temp =  mongoose.model('test' , new mongoose.Schema( {
    a: 
        new mongoose.Schema({
            b:  new mongoose.Schema({})
    })
}));
m2s(temp);
```

**Comment by blugavere** [comment]
published fix @ 1.0.2

**Comment by erezz33** [comment]
Thanks for the fix. There is no error but the functinality is not working for this case.
for instance:
const m2s = require('mongoose-to-swagger');
const temp =  mongoose.model('test' , new mongoose.Schema( {   
    a: 
        new mongoose.Schema({
            b: {
                type :  new mongoose.Schema({
                    c : String,
                    d :  new mongoose.Schema({})
                })
            }
    })
}));
m2s(temp);
console.log(JSON.stringify(m2s(temp)));

// part of the response
{"title":"test","properties":{"a":{"type":"object","properties":{"b":{"type":"object","**properties":{}**}

** as you can see b is missing properties defined in it's mongoose schema (c and d)

**Comment by blugavere** [comment]
@erezz33 thanks for pointing out - pushed fix to 1.0.3 if you want to give it another go

**Comment by erezz33** [comment]
thanks a lot, works perfect!
